packages/kotti-ui/source/kotti-field/KtField.vue

```diff
@@ -351,7 +351,7 @@ export default defineComponent<{
 
 			.yoco {
 				display: flex;
-				font-size: 1.1em;
+				font-size: 1.4em;
```

Comment:
This also modifies the right & left icons:

![image](https://user-images.githubusercontent.com/1133858/114391533-e5a38d00-9b97-11eb-836e-b774c2f6a5d7.png)

Is this intentional?

Reply:
no